esp/bindings/http/platform/httpservice.cpp

```diff
@@ -312,6 +313,10 @@ int CEspHttpServer::processRequest()
                     return onGetNavEvent(m_request.get(), m_response.get());
                 else if (!stricmp(methodName.str(), "soapreq"))
                     return onGetBuildSoapRequest(m_request.get(), m_response.get());
+                else if (strieq(methodName.str(), "createjob")) //Is this a good place to create a job and get job status?
```

Comment:
@wangkx this looks like someone would just create a job externally, by calling this URL/method?  I was expecting the job to be created as a side effect of calling something else, like creating a zap report, or in my case publishing something that needs to copy files?

Reply:
Would break what a browser is doing if ESP creates the job AND returns the job ID while both ESP AND browser are working on some jobs, like downloading, uploading, etc?